onos-classic/templates/configmap-probe.yaml

```diff
@@ -13,6 +13,26 @@ data:
     #!/bin/bash
     set -e
     host=$(hostname -s)
-    config=$(curl -s http://$host:8181/onos/v1/cluster/$host --user onos:rocks)
-    echo $config
-    printf '%q' $config | grep -q "READY"
+    result=$(curl -s -f http://$host:8181/onos/v1/cluster/$host --user onos:rocks)
+    echo $result
+
+    printf '%q' $result | grep -q "READY"
+    if [ ! $? ]; then
+      echo "Cluster is not yet ready"
+      return -1
+    fi
+
+    apps=()
+    {{- range .Values.apps }}
+    apps+=("{{ . }}")
+    {{- end }}
+
+    for app in ${apps[@]}; do
+      result=$(curl -s -f http://$host:8181/onos/v1/applications/$app --user onos:rocks)
```

Comment:
you don't want to hard code credentials.

Reply:
This helm chart doesn't support changing login credentials anyway. It will be more meaningful if we implement that as well. I'd leave this to a separate patch.